src/watson/urls.py

```diff
@@ -2,13 +2,13 @@
 
 from __future__ import unicode_literals
 
-from django.conf.urls import url, patterns
+from django.conf.urls import url
+from . import views
```

Comment:
Heyo thanks for all the work! I'm curious about this, why no relative imports? Seems to me to be the "best practice" way of doing things?